operator/internal/manifests/var.go

```diff
@@ -135,6 +135,16 @@ func commonLabels(stackName string) map[string]string {
 	}
 }
 
+func componentInstaceLabels(l labels.Set) map[string]string {
+	// This code assumes that this function will never be called with a set of labels
+	// that don't have the "component" and "instance" labels since we enforce those on
+	// all the components of the LokiStack
+	return map[string]string{
+		kubernetesInstanceLabel:  l[kubernetesInstanceLabel],
+		kubernetesComponentLabel: l[kubernetesComponentLabel],
```

Comment:
It's much better to have the signature be very clear than use a map for parameters.
```suggestion
func componentInstaceLabels(component string, stackName string) map[string]string {
	return map[string]string{
	        kubernetesInstanceLabel:  stackName,
		kubernetesCompomentLabel: component,
					
```